spec/api-callbacks-registry-spec.js

```diff
@@ -12,37 +17,37 @@ describe('CallbacksRegistry module', () => {
     const cb = () => [1, 2, 3, 4, 5]
     const key = registry.add(cb)
 
-    assert.exists(key)
+    expect(key).to.exist()
   })
 
   it('returns a specified callback if it is in the registry', () => {
     const cb = () => [1, 2, 3, 4, 5]
     const key = registry.add(cb)
     const callback = registry.get(key)
 
-    assert.equal(callback.toString(), cb.toString())
+    expect(callback.toString()).equal(cb.toString())
   })
 
   it('returns an empty function if the cb doesnt exist', () => {
     const callback = registry.get(1)
 
-    assert.isFunction(callback)
+    expect(callback).to.be.a('function')
   })
 
   it('removes a callback to the registry', () => {
     const cb = () => [1, 2, 3, 4, 5]
     const key = registry.add(cb)
 
-    assert.exists(key)
+    expect(key).to.exist()
 
     const beforeCB = registry.get(key)
 
-    assert.equal(beforeCB.toString(), cb.toString())
+    expect(beforeCB.toString()).to.equal(cb.toString())
```

Comment:
This check and the one right above are redundant. We already have separate tests for those.